docs/documentation.rst

```diff
@@ -0,0 +1,46 @@
+Generating the documentation locally
+----------------------------------
+
+Install the required packages
+
+.. code-block:: bash
+
+   $ pip install sphinx sphinx-autobuild sphinxcontrib-napoleon
+
+.. tip::
+   If you don't have sudo privileges, you can use the ``-- user`` flag to 
+   install the required packages locally.
+
+Enter in the docs directory and rename the existing conf.py
```

Comment:
Why do you do this? I've always been able to build the documentation without doing any of this.